Digital Shadows/digital_shadows_searchlight/ingest/parser.yml

```diff
@@ -0,0 +1,26 @@
+name: digital_shadows_searchlight
+pipeline:
+  - external:
+        name: json.parse-json
+        properties:
+            input_field: '{{original.message}}'
+            output_field: message
+    name: json_event
+  - name: field_extraction
+stages:
+  field_extraction:
+    actions:
+      - set:
+          event.id: "{{json_event.message.id}}"
```

Comment:
not sure you should override this field. @otetard can you confirm this field is automatically set by SEKOIA

Reply:
Replaced by `digital_shadows_searchlight.event.id`